manifests/m/Mariocube/EasyCSM/1.4/Mariocube.EasyCSM.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Mariocube.EasyCSM
+PackageVersion: 1.4

```

```suggestion
PackageVersion: 1.4
AppsAndFeaturesEntries:
  - DisplayVersion:
```
The installer doesn't write a display version string to registry